articles/how-to-fix-and-prevent-sql-injection-in-wordpress/index.md

```diff
@@ -0,0 +1,194 @@
+~~~~
+Manipulate the string in the code to determine when any of the users found were created.
+
+Step 4: Delete any rogue users using the command below:
+
+~~~~sql
+DROP USER 'malicious'@'localhost';
+~~~~
+
+Step 5: Change and secure your passwords using the command below.
+~~~~sql
+UPDATE users SET pass = concat('ZZZ', sha(concat(pass, md5(rand()))));
+~~~~
+
+##### Fix WordPress SQLi Attack: Vital updates
+
+Updates for your WordPress site are very vital in keeping ahead of day to day vulnerabilities. Make sure to update your WordPress Core files oftenly as well as your theme and plugins. Avoid using nulled themes on your WP site. In addition, make sure to use reputable plugins on your site.
+
+##### Fix WordPress SQLi Attack: Use a Firewall
+
+A firewall detects and shields your website from SQLi as well other attacks depending on the features of your firewall service provider. There are quite of firewall security solutions on the internet; look out for one that suits your WP security needs.
+
+##### Fix WordPress SQLi Attack: Pentesting
+
+[Pentesting/penetration testing](https://www.cloudflare.com/learning/security/glossary/what-is-penetration-testing/) or rather website security audit is a vital operation to keep your WP site away from possible SQLi attacks. WordPress Pentesting involves doing simulated SQLi attacks on your site to make all loopholes are covered. This process ensures all loopholes are found and sealed before an attacker finds and takes advantage of the same.
+
+### How to Prevent SQLi in WordPress
+
+As they say, prevention is better than cure. Don't wait to prove if that is right or wrong with a successful SQLi on your WordPress website. Proper site maintenance practices will save you from SQLi attacks. These practices include regular updates for WordPress core files, plugins updates, use of reputable plugins, avoid nulled themes and minimal user inputs fields as much as possible.
+
+Normalize use of ready-made commands for your database operations. WordPress has prepared statements for database operations which are well structured to mitigate risk on your website. Ensure your website is enabled for context data filtering such as only allowing digits for phone number fields.
+
+##### Use Secure WordPress themes &amp; Plugins
+
+Themes and plugins are unavoidable while using the WordPress CMS hence they are the silver lining to the open source project. Nevertheless, ensure you are using reputable plugins and themes from the WordPress repository. If you are going to use premium plugins and themes you want to look for such at top rated repositories such as [Envato Market.](https://envato.com/)
+
+Be more vigilant with contact form and subscription plugins as they pose user input threat to your site.
+
+##### Mask your WordPress version
+
+This may seem nagging but you may want to give hackers a hard time cracking your website. To hide your WP version, use the code below;
```

Comment:
Another way to give hackers a hard time cracking your website is to hide your WP version. To do so, use the code below:

This would flow better.